code/modules/procedural_mapping/mapGenerators/cellular.dm

```diff
@@ -75,12 +75,12 @@
 					value += edge_value
 					continue
 			value += old_state[n_x][n_y]
-	value -= old_state[i][j]
+	value -= old_state[index_y][index_y]
```

Comment:
```suggestion
	value -= old_state[y][y]
```

Also this line is bugged. you use y in both but the old was i and j, so one of those should be an x.